pages/news/2020-XX-XX-concrete-action.html

```diff
@@ -36,6 +36,7 @@
 ## Building A More Diverse Community
 
 In our discussions, we quickly noted that the Mixxx contributor base is largely white, male, and either from the US or Western Europe.
+This is especially ironic considering that the art, techniques, and culture of DJing were pioneered by people of color.
```

Comment:
Did you intentionally omit the link to https://www.dukeupress.edu/Love-Saves-the-Day/ ? If you have a link to another source about the history of DJing which is more succinct, that would work too.

Reply:
Maybe even mention Grandmaster Flash and Grand Wizard Theodore?

Reply:
@ThisGrrrlFriday recommends that we avoid trying to make this blog post into a history lesson.  The history of dance music is too complex to sum up in a single sentence.  It's enough to say that our contributors have been homogenious.

Reply:
I don't think that our contributors are homogenious, but people who are not white cis-males are underrepresented.

Reply:
restored sentence